# Awesome De novo Drug Design Papers :mute:

![heey](https://img.shields.io/badge/Be%20quite!-Someone%20reading!-blue)

Papers about **De novo Drug Design :pill:** collected by [@HUBioDataLab](https://github.com/HUBioDataLab) members.

**Please feel free to add good, related papers. If there is any error about links, don't hesitate to pull!**

Template by [@mengliu1998](https://github.com/mengliu1998)!

2022
----
* [arXiv 2022] **Structure-based drug discovery with deep learning** *Review*  [[Paper]](https://arxiv.org/abs/2212.13295) 
* [arXiv 2022] **DiffLinker: Equivariant 3D-Conditional Diffusion Model for Molecular Linker Design**  [[Paper]](https://arxiv.org/abs/2210.05274) [[Code]](https://github.com/igashov/DiffLinker)
* [arXiv 2022] **DiffDock: Diffusion Steps, Twists, and Turns for Molecular Docking**  [[Paper]](https://arxiv.org/abs/2210.01776) [[Code]](https://github.com/gcorso/diffdock)
* [ACS JCIM 2022] **HyFactor: A Novel Open-Source, Graph-Based Architecture for Chemical Structure Generation** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00744) [[Code]](https://github.com/Laboratoire-de-Chemoinformatique/HyFactor)
* [Journal of Cheminformatics 2022] **Designing optimized drug candidates with Generative Adversarial Network** [[Paper]](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-022-00623-6) [[Code]](https://github.com/larngroup/GAN-Drug-Generator)
* [arXiv 2022] **A Survey on Deep Graph Generation: Methods and Applications**  [[Paper]](https://arxiv.org/abs/2203.06714)
* [Nature Machine Intelligence 2022] **A deep generative model enables automated structure elucidation of novel psychoactive substances** [[Paper]](https://www.nature.com/articles/s42256-021-00407-x) [[Code]](https://github.com/skinnider/NPS-generation)
* [arXiv 2022] **Equivariant Diffusion for Molecule Generation in 3D** [[Paper]](https://arxiv.org/abs/2203.17003) [[Code]](https://github.com/ehoogeboom/e3_diffusion_for_molecules)
* [arXiv 2022] **Top-N: Equivariant set and graph generation without exchangeability** [[Paper]](https://arxiv.org/abs/2110.02096) [[Code]](https://github.com/cvignac/top-n)
* [ACS JCIM 2022] **RetroGNN: Fast Estimation of Synthesizability for Virtual Screening and De Novo Design by Learning from Slow Retrosynthesis Software** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.1c01476) 
* [ChemRxiv 2022] **Conditional ùõΩ-VAE for De Novo Molecular Generation** [[Paper]](https://chemrxiv.org/engage/chemrxiv/article-details/626b4332368ab64701913771)

2021
----
* [scientific report 2021] **Transformer neural network for protein-specific de novo drug generation as a machine translation problem** [[Paper]](https://www.nature.com/articles/s41598-020-79682-4)[[Code]](https://github.com/dariagrechishnikova/molecule_structure_generation)
* [RSC 2021] **Attention-based generative models for de novo molecular design**  [[Paper]](https://pubs.rsc.org/en/content/articlehtml/2021/sc/d1sc01050f)[[Code]](https://github.com/oriondollar/TransVAE)
* [Nature Communications 2021] **Deep generative neural network for accurate drug response imputation** [[Paper]](https://www.nature.com/articles/s41467-021-21997-5)[[Code]](https://github.com/bsml320/VAEN/)
* [Journal of Molecular Modeling 2021] **Generative chemistry: drug discovery with deep learning generative models** [[Paper]](https://arxiv.org/abs/2008.09000)
* [arXiv 2021] **A Graph VAE and Graph Transformer Approach to Generating Molecular Graphs** [[Paper]](https://arxiv.org/abs/2104.04345)
* [Journal of Chemical Information and Modeling 2021] **OpenChem: A Deep Learning Toolkit for Computational Chemistry and Drug Design** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.0c00971)[[Code]](https://github.com/Mariewelt/OpenChem)
* [arXiv 2021] **Transformers for Molecular Graph Generation** [[Paper]](https://www.esann.org/sites/default/files/proceedings/2021/ES2021-112.pdf)[[Code]](https://gitlab.uni-oldenburg.de/gies6280/molegent)
* [ACS JCIM 2021] **Exploring Graph Traversal Algorithms in Graph-Based Molecular Generation** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.1c00777)[[Code]](https://github.com/MolecularAI/GraphINVENT)
* [Nature 2021] **Synthon-based ligand discovery in virtual libraries of over 11 billion compounds** [[Paper]](https://www.nature.com/articles/s41586-021-04220-9)
* [ACS JCIM 2021] **Comparative Study of Deep Generative Models on Chemical Space Coverage** [[Paper]](https://doi.org/10.1021/acs.jcim.0c01328) 
* [ACS JCIM 2021] **Generative Chemical Transformer: Neural Machine Learning of Molecular Geometric Structures from Chemical Language via Attention** [[Paper]](https://doi.org/10.1021/acs.jcim.1c01289)
* [ACS JCIM 2021] **De Novo Structure-Based Drug Design Using Deep Learning** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.1c01319) 



2020
----

* [Royal Society of Chemistry 2020] **Artificial Intelligence in Drug Discovery** 
* [RSC 2020] **Beyond Generative Models: Superfast Traversal, Optimization, Novelty, Exploration and Discovery (STONED) Algorithm for Molecules using SELFIES** [[Paper]](https://pubs.rsc.org/en/content/articlelanding/2021/sc/d1sc00231g)[[Code]](https://github.com/aspuru-guzik-group/stoned-selfies)
* [ChemRxiv 2020] **Comparative Study of Deep Generative Models on Chemical Space Coverage** [[Paper]](https://chemrxiv.org/engage/chemrxiv/article-details/60c755389abda285f4f8e2d1)[[Code]](https://github.com/jeah-z/Generative_Models_benchmark_gdb13)
* [CUoT 2020] **Comparison of State-of-the-art Algorithms for de novo Drug Design** [[Paper]](https://odr.chalmers.se/bitstream/20.500.12380/301739/1/CSE%2020-69%20Sundkvist%20Nilsson.pdf)[[Code]](https://github.com/sebastiandro/de-novo-evaluation)
* [arXiv 2020] **Deep Molecular Dreaming: Inverse machine learning for de-novo molecular design and interpretability with surjective representations** [[Paper]](https://www.arxiv-vanity.com/papers/2012.09712/)
* [Nature Machine Intelligence 2020] **Direct steering of de novo molecular generation with descriptor conditional recurrent neural networks** [[Paper]](https://www.nature.com/articles/s42256-020-0174-5)[[Code]](https://github.com/pcko1/Deep-Drug-Coder/tree/master/datasets)
* [arxiv 2020] **Generating 3D Molecular Structures Conditional on a Receptor Binding Site with Deep Generative Models** [[Paper]](https://arxiv.org/abs/2010.14442)
* [Drug Discovery Today: Technologies 2020] **Graph-based generative models for de Novo drug design** [[Paper]](https://www.sciencedirect.com/science/article/pii/S1740674920300251)
* [arXiv 2020] **MolDesigner: Interactive Design of Efficacious Drugs with Deep Learning** [[Paper]](https://arxiv.org/abs/2010.03951)
* [ChemRxiv 2020] **Practical Notes on Building Molecular Graph Generative Models** [[Paper]](https://chemrxiv.org/engage/chemrxiv/article-details/60c74f55567dfe705bec5672)[[Code]](https://github.com/MolecularAI/GraphINVENT)
* [arXiv 2020] **RetroGNN: Approximating Retrosynthesis by Graph Neural Networks for De Novo Drug Design** [[Paper]](https://arxiv.org/abs/2011.13042)
* [arXiv 2020] **Scalable Deep Generative Modeling for Sparse Graphs** [[Paper]](https://arxiv.org/abs/2006.15502)[[Code]](https://github.com/google-research/google-research/tree/master/bigg)
* [arXiv 2020] **Seq2Mol: Automatic design of de novo molecules conditioned by the target protein sequences through deep neural networks** [[Paper]](https://arxiv.org/abs/2010.15900)
* [arXiv 2020] **Target-specific and selective drug design for covid-19 using deep generative models** [[Paper]](https://arxiv.org/abs/2004.01215)
* [ChemRxiv 2020] **REINVENT 2.0 ‚Äì an AI Tool for De Novo Drug Design** [[Paper]](https://chemrxiv.org/engage/chemrxiv/article-details/60c74f75bdbb89eaf7a39d8a)[[Code]](https://github.com/MolecularAI/Reinvent)
* [Front. Pharmacol. 2020] **Molecular Generation for Desired Transcriptome Changes With Adversarial Autoencoders** [[Paper]](https://www.frontiersin.org/articles/10.3389/fphar.2020.00269/full)[[Code]](https://github.com/insilicomedicine/BiAAE)
* [ACS 2020] **The Synthesizability of Molecules Proposed by Generative Models** [[Paper]](https://pubs.acs.org/doi/full/10.1021/acs.jcim.0c00174)[[Code]](https://github.com/wenhao-gao/askcos_synthesizability)

2019
----

* [Journal of Cheminformatics 2019] **A de novo molecular generation method using latent vector based generative adversarial network**  [[Paper]](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-019-0397-9)[[Code]](https://github.com/Dierme/latent-gan)
* [bioRxiv 2019] **Accelerating Protein Design Using Autoregressive Generative Models** [[Paper]](https://www.biorxiv.org/content/10.1101/757252v1)[[Code]](https://github.com/debbiemarkslab/SeqDesign)
* [Nature Machine Intelligence 2019] **Automated de novo molecular design by hybrid machine intelligence and rule-driven chemical synthesis** [[Paper]](https://www.nature.com/articles/s42256-019-0067-7)
* [arXiv 2019] **Black Box Recursive Translations for Molecular Optimization** [[Paper]](https://arxiv.org/abs/1912.10156)
* [arXiv 2019] **ChemBO: Bayesian Optimization of Small Organic Molecules with Synthesizable Recommendations** [[Paper]](https://arxiv.org/abs/1908.01425)[[PapersWithCode]](https://paperswithcode.com/paper/chembo-bayesian-optimization-of-small-organic)
* [Bioinformatics 2019] **Computational modeling of cellular structures using conditional deep generative networks** [[Paper]](https://academic.oup.com/bioinformatics/article/35/12/2141/5162747)[[Code]](https://github.com/divelab/cgan/)
* [Journal of Chemical Information and Modeling 2019] **Conditional Molecular Design with Deep Generative Models** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.8b00263)[[Code]](https://github.com/nyu-dl/conditional-molecular-design-ssvae)
* [Journal of Chemical Information and Modeling 2019] **De Novo Molecular Design by Combining Deep Autoencoder Recurrent Neural Networks with Generative Topographic Mapping** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.8b00751)
* [bioRxiv 2019] **Decomposing Retrosynthesis into Reactive Center Prediction and Molecule Generation** [[Paper]](https://www.biorxiv.org/content/10.1101/677849v1.full)
* [arXiv 2019] **Deep learning for molecular design‚Äîa review of the state of the art** [[Paper]](https://arxiv.org/abs/1903.04388)
* [Future medicinal chemistry 2019] **Deep learning for molecular generation** [[Paper]](https://pubmed.ncbi.nlm.nih.gov/30698019/)
* [Journal of Chemical Information and Modeling 2019] **Deep Learning in Chemistry** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.9b00266)
* [Journal of chemical information and modeling 2019] **Drug Analogs from Fragment-Based Long Short-Term Memory Generative Neural Networks** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.8b00902)
* [RSC 2019] **Efficient Multi-Objective Molecular Optimization in a Continuous Latent Space** [[Paper]](https://pubs.rsc.org/en/content/articlelanding/2019/sc/c9sc01928f)[[Code]](https://github.com/jrwnter/mso)
* [Journal of Cheminformatics 2019] **Exploring the GDB-13 chemical space using deep generative models** [[Paper]](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-019-0341-z)[[Code]](https://github.com/undeadpixel/reinvent-gdb13)
* [Advances in Neural Information Processing Systems 2019] **Generative models for graph-based protein design** [[Paper]](https://papers.nips.cc/paper/2019/hash/f3a4ff4839c56a5f460c88cce3666a2b-Abstract.html)[[Code]](https://github.com/jingraham/neurips19-graph-protein-design)
* [bioRxiv 2019] **Improving protein function prediction with synthetic feature samples created by generative adversarial networks** [[Paper]](https://www.biorxiv.org/content/10.1101/730143v1)[[Code]](https://github.com/psipred/FFPredGAN)
* [arXiv 2019] **Likelihood-Free Inference and Generation of Molecular Graphs** [[Paper]](https://arxiv.org/pdf/1905.10310.pdf)[[Code]](https://github.com/ai-med/almgig)
* [arXiv 2019] **A Model to Search for Synthesizable Molecules** [[Paper]](https://arxiv.org/abs/1906.05221)[[Code]](https://github.com/john-bradshaw/molecule-chef)
* [ACS Central Science 2019] **Molecular transformer: a model for uncertainty-calibrated chemical reaction prediction** [[Paper]](https://pubs.acs.org/doi/10.1021/acscentsci.9b00576)[[Code]](https://github.com/pschwllr/MolecularTransformer)
* [arXiv 2019] **MolecularRNN: Generating realistic molecular graphs with optimized properties** [[Paper]](https://arxiv.org/abs/1905.13372)[[PapersWithCode]](https://paperswithcode.com/paper/molecularrnn-generating-realistic-molecular)
* [ChemRxiv 2019] **Multi-Resolution Autoregressive Graph-to-Graph Translation for Molecules** [[Paper]](https://chemrxiv.org/engage/chemrxiv/article-details/60c74268469df460ccf42f99)
* [Journal of Computer-Aided Molecular Design 2019] **Multi-task generative topographic mapping in virtual screening** [[Paper]](https://pubmed.ncbi.nlm.nih.gov/30739238/)
* [Journal of Cheminformatics 2019] **Randomized SMILES Strings Improve the Quality of Molecular Generative Models** [[Paper]](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-019-0393-0)[[Code]](https://github.com/molecularsets/moses)
* [arXiv 2019] **Scaffold-based molecular design using graph generative model** [[Paper]](https://arxiv.org/abs/1905.13639)
* [ACS 2019] **Shape-Based Generative Modeling for de Novo Drug Design** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.8b00706)[[Code]](https://github.com/compsciencelab/ligdream)
* [arXiv 2019] **A Two-Step Graph Convolutional Decoder for Molecule Generation** [[Paper]](https://arxiv.org/abs/1906.03412)

2018
----

* [ACS Central Science 2018] **Automatic Chemical Design Using a Data-Driven Continuous Representation of Molecules** [[Paper]](https://pubs.acs.org/doi/10.1021/acscentsci.7b00572)[[Code]](https://github.com/aspuru-guzik-group/chemical_vae)
* [Science Advances 2018] **Deep reinforcement learning for de novo drug design** [[Paper]](https://www.science.org/doi/10.1126/sciadv.aap7885)[[Code]](https://github.com/isayev/ReLeaSE)
* [Journal of Chemical Information and Modeling 2018] **Fr√©chet ChemNet Distance: A Metric for Generative Models for Molecules in Drug Discovery** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.8b00234)[[Code]](https://github.com/bioinf-jku/FCD)
* [Proceedings of the AAAI conference on artificial intelligence 2018] **Graphgan: Graph representation learning with generative adversarial nets** [[Paper]](https://arxiv.org/abs/1711.08267)[[Code]](https://github.com/hwwang55/GraphGAN)
* [arXiv 2018] **Learning deep generative models of graphs** [[Paper]](https://arxiv.org/abs/1803.03324)[[PapersWithCode]](https://paperswithcode.com/paper/learning-deep-generative-models-of-graphs)
* [arXiv 2018] **MolGAN: An implicit generative model for small molecular graphs** [[Paper]](https://arxiv.org/abs/1805.11973)[[PapersWithCode]](https://paperswithcode.com/paper/molgan-an-implicit-generative-model-for-small)
* [Journal of Cheminformatics 2018] **Multi-objective de novo drug design with conditional graph generative model** [[Paper]](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-018-0287-6)[[Code]](https://github.com/kevinid/molecule_generator)
* [Nature 2018] **Planning chemical syntheses with deep neural networks and symbolic AI** [[Paper]](https://www.nature.com/articles/nature25978)
* [ACS Medicinal Chemistry Letters 2018] **Transforming Computational Drug Discovery with Machine Learning and AI** [[Paper]](https://pubs.acs.org/doi/10.1021/acsmedchemlett.8b00437)
* [Journal of chemical information and modeling 2018] **Sparse Generative Topographic Mapping for Both Data Visualization and Clustering** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.8b00528)[[Code]](https://github.com/hkaneko1985/gtm-generativetopographicmapping)
* [ACL Anthology 2018] **Bidirectional Generative Adversarial Networks for Neural Machine Translation** [[Paper]](https://aclanthology.org/K18-1019/)
* [KDD 2018] **Learning Deep Network Representations with Adversarially
Regularized Autoencoders** [[Paper]](https://dl.acm.org/doi/pdf/10.1145/3219819.3220000)
* [arXiv 2018] **NetGAN: Generating Graphs via Random Walks** [[Paper]](https://arxiv.org/abs/1803.00816) [[Code]](https://github.com/danielzuegner/netgan)
* [Journal of Cheminformatics 2018] **Molecular generative model based on conditional variational autoencoder for de novo molecular design** [[Paper]](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-018-0286-7) [[Code]](https://github.com/jaechanglim/CVAE)


2017
----

* [Molecular Pharmaceutics 2017] **druGAN: An Advanced Generative Adversarial Autoencoder Model for de Novo Generation of New Molecules with Desired Molecular Properties in Silico** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.molpharmaceut.7b00346)
* [ACS Central Science 2017] **Generating Focused Molecule Libraries for Drug Discovery with Recurrent Neural Networks** [[Paper]](https://pubs.acs.org/doi/10.1021/acscentsci.7b00512)
* [Journal of Cheminformatics 2017] **Molecular de-novo design through deep reinforcement learning** [[Paper]](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-017-0235-x)[[Code]](https://github.com/MarcusOlivecrona/REINVENT)
* [arXiv 2017] **GraphGAN: Graph Representation Learning with Generative Adversarial Nets** [[Paper]](https://arxiv.org/abs/1711.08267) [[Code]](https://github.com/hwwang55/GraphGAN)

