# Awesome De novo Drug Design Papers :mute:

![heey](https://img.shields.io/badge/Be%20quite!-Someone%20reading!-blue)

Papers about **De novo Drug Design :pill:**

**Please feel free to add good, related papers. If there is any error about links, don't hesitate to pull!**


2023
----
* [AAAI 2023] **MDM: Molecular Diffusion Model for 3D Molecule Generation** [[Paper]](https://ojs.aaai.org/index.php/AAAI/article/view/25639) [[Code]](https://github.com/tencent-ailab/MDM)
* [ChemRxiv 2023] **MoFlowGAN: Combining adversarial and likelihood learning for targeted molecular generation** [[Paper]](https://chemrxiv.org/engage/chemrxiv/article-details/6495bd352e632767b0ab56be) [[Code]](https://github.com/thisisntnathan/MoFlowGAN)
* [arXiv 2023] **Graph Generative Model for Benchmarking Graph Neural Networks** [[Paper]](https://arxiv.org/abs/2207.04396) [[Code]](https://github.com/minjiyoon/CGT)
* [bioRxiv 2023] **Target-aware Variational Auto-encoders for Ligand Generation with Multimodal Protein Representation Learning** [[Paper]](https://www.biorxiv.org/content/10.1101/2023.08.10.552868v1.abstract) [[Code]](https://github.com/HySonLab/Ligand_Generation)
* [arXiv 2023] **Learning Joint 2D & 3D Diffusion Models for Complete Molecule Generation** [[Paper]](https://arxiv.org/abs/2305.12347) [[Code]](https://github.com/graph-0/jodo)
* [arXiv 2023] **Molecule Design by Latent Prompt Transformer** [[Paper]](https://arxiv.org/abs/2310.03253) 
* [Journal of Cheminformatics 2023] **ScaffoldGVAE: scaffold generation and hopping of drug molecules via a variational autoencoder based on multi-view graph neural networks** [[Paper]](https://link.springer.com/article/10.1186/s13321-023-00766-0) [[Code]](https://github.com/ecust-hc/ScaffoldGVAE)
* [Journal of Cheminformatics 2023] **MolFilterGAN: a progressively augmented generative adversarial network for triaging AI-designed molecules** [[Paper]](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-023-00711-1) [[Code]](https://github.com/MolFilterGAN/MolFilterGAN)
* [arXiv 2023] **Reinforcement Learning for Generative AI: State of the Art, Opportunities and Open Research Challenges** [[Paper]](https://arxiv.org/abs/2308.00031) 
* [ICML 2023] **Coarse-to-Fine: a Hierarchical Diffusion Model for Molecule Generation in 3D** [[Paper]](https://proceedings.mlr.press/v202/qiang23a.html) [[Code]](https://github.com/qiangbo1222/HierDiff)
* [ACS JCIM 2023] **De Novo Generation of Chemical Structures of Inhibitor and Activator Candidates for Therapeutic Target Proteins by a Transformer-Based Variational Autoencoder and Bayesian Optimization** [[Paper]](https://pubs.acs.org/doi/full/10.1021/acs.jcim.3c00824) [[Code]](https://yamanishi.cs.i.nagoya-u.ac.jp/triompheboa/)
* [ECML PKDD 2023] **Target-Aware Molecular Graph Generation** [[Paper]](https://link.springer.com/chapter/10.1007/978-3-031-43427-3_25)
* [ECML PKDD 2023] **SpotGAN: A Reverse-Transformer GAN Generates Scaffold-Constrained Molecules with Property Optimization** [[Paper]](https://link.springer.com/chapter/10.1007/978-3-031-43412-9_19) [[Code]](https://github.com/naruto7283/SpotGAN)
* [Journal of Cheminf. 2023] **Conditional reduction of the loss value versus reinforcement learning for biassing a de-novo drug design generator** [[Paper]](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-022-00643-2) [[Code]](https://github.com/amine179/DrugDesign)
* [ACS JCIM 2023] **Exploring the Advantages of Quantum Generative Adversarial Networks in Generative Chemistry** [[Paper]](https://pubs.acs.org/doi/full/10.1021/acs.jcim.3c00562) [[Code]](https://github.com/pykao/QuantumMolGAN-PyTorch)
* [Research Square 2023] **LS-MolGen: Ligand-and-Structure Dual-driven Deep Reinforcement Learning for Target-specific Molecular Generation Improves Binding Affinity and Novelty** [[Paper]](https://www.researchsquare.com/article/rs-2793302/v1) [[Code]](https://github.com/songleee/LS-MolGen)
* [bioRxiv 2023] **An Equivariant Generative Framework for Molecular Graph-Structure Co-Design** [[Paper]](https://www.biorxiv.org/content/10.1101/2023.04.13.536803v1.abstract) [[Code]](https://github.com/zaixizhang/MolCode)
* [ChemRxiv 2023] **De Novo Design of Molecules Towards Biased Properties via a Deep Generative Framework and Iterative Transfer Learning** [[Paper]](https://chemrxiv.org/engage/chemrxiv/article-details/643019ee736114c963130d96) [[Code]](https://github.com/linresearchgroup/RRCGAN_Molecules)
* [Bioinformatics 2023] **De novo drug design by iterative multiobjective deep reinforcement learning with graph-based molecular quality assessment** [[Paper]](https://academic.oup.com/bioinformatics/article/39/4/btad157/7085596) [[Code]](https://github.com/yifang000/QADD)
* [arXiv 2023] **Balancing Exploration and Exploitation: Disentangled Œ≤-CVAE in De Novo Drug Design** [[Paper]](https://arxiv.org/abs/2306.01683)
* [ACS JCIM 2023] **Generative Models Should at Least Be Able to Design Molecules That Dock Well: A New Benchmark** [[Paper]](https://pubs.acs.org/doi/full/10.1021/acs.jcim.2c01355) [[Code]](https://github.com/cieplinski-tobiasz/smina-docking-benchmark)
* [Molecules 2023] **cMolGPT: A Conditional Generative Pre-Trained Transformer for Target-Specific De Novo Molecular Generation** [[Paper]](https://www.mdpi.com/1420-3049/28/11/4430) [[Code]](https://github.com/VV123/cMolGPT)
* [ChemRxiv 2023] **A Protein-Ligand Interaction-focused 3D Molecular Generative Framework for Generalizable Structure-based Drug Design** [[Paper]](https://chemrxiv.org/engage/chemrxiv/article-details/6482d9dbbe16ad5c57af1937) [[Code]](https://github.com/ACE-KAIST/DeepICL)
* [arXiv 2023] **SILVR: Guided Diffusion for Molecule Generation** [[Paper]](https://arxiv.org/abs/2304.10905) [[Code]](https://github.com/ehoogeboom/e3_diffusion_for_molecules)
* [Journal of Chem. 2023] **Deep generative model for drug design from protein target sequence** [[Paper]](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-023-00702-2) [[Code]](https://github.com/viko-3/TargetGAN)
* [Journal of Mol. Mod. 2023] **De novo drug design based on Stack-RNN with multi-objective reward-weighted sum and reinforcement learning** [[Paper]](https://link.springer.com/article/10.1007/s00894-023-05523-6) [[Code]](https://github.com/PengWeiHu1/mul_RL/tree/master)
* [ACS 2023] **Machine Learning for Fast, Quantum Mechanics-Based Approximation of Drug Lipophilicity** [[Paper]](https://pubs.acs.org/doi/full/10.1021/acsomega.2c05607) [[Code]](https://github.com/cisert/rescoss_logp_ml)
* [bioRxiv 2023] **Variational graph encoders: a surprisingly effective generalist algorithm for holistic computer-aided drug design** [[Paper]](https://www.biorxiv.org/content/10.1101/2023.01.11.523575v2.abstract) [[Code]](https://github.com/Chokyotager/NotYetAnotherNightshade)
* [ChemRxiv 2023] **ChemTSv2: Democratizing Functional Molecular Design Using de novo Molecule Generator** [[Paper]](https://chemrxiv.org/engage/chemrxiv/article-details/63da180001ecc690f91b0c78) [[Code]](https://github.com/molecule-generator-collection/ChemTSv2)
* [arXiv 2023] **Generative Diffusion Models on Graphs: Methods and Applications** [[Paper]](https://arxiv.org/abs/2302.02591)
* [bioRxiv 2023] **A dual diffusion model enables 3D binding bioactive molecule generation and lead optimization given target pockets** [[Paper]](https://www.biorxiv.org/content/10.1101/2023.01.28.526011v1.abstract) [[Data]](https://bits.csb.pitt.edu/files/crossdock2020/)
* [arXiv 2023] **De Novo Molecular Generation via Connection-aware Motif Mining** [[Paper]](https://arxiv.org/abs/2302.01129) [[Code]](https://github.com/miralab-ustc/ai4sci-micam)
* [ICLR 2023] **3D Equivariant Diffusion for Target-Aware Molecule Generation and Affinity Prediction** [[Paper]](https://arxiv.org/abs/2303.03543) [[Code]](https://github.com/guanjq/targetdiff)
* [ScienceDirect 2023] **Structure-based drug design with geometric deep learning** [[Paper]](https://www.sciencedirect.com/science/article/pii/S0959440X23000222)
* [ACS 2023] **Chemistry42: An AI-Driven Platform for Molecular Design and Optimization** [[Paper]](https://pubs.acs.org/doi/full/10.1021/acs.jcim.2c01191) [[Code]](https://github.com/insilicomedicine/GENTRL) [[Platform]](https://chemistry42.com)
* [Journal of Cheminformatics 2023] **DrugEx v3: scaffold-constrained drug design with graph transformer-based reinforcement learning** [[Paper]](https://link.springer.com/article/10.1186/s13321-023-00694-z) [[Code]](https://github.com/CDDLeiden/DrugEx)
* [arXiv 2023] **Target Specific De Novo Design of Drug Candidate Molecules with Graph Transformer-based Generative Adversarial Networks** [[Paper]](https://arxiv.org/abs/2302.07868) [[Code]](https://github.com/asarigun/DrugGEN)
* [Wiley 2023] **De novo design of a molecular catalyst using a generative model** [[Paper]](https://onlinelibrary.wiley.com/doi/10.1002/anie.202218565) [[Code]](https://github.com/jensengroup/mbh_catalyst_ga)
* [ScienceDirect 2023] **DNMG: Deep molecular generative model by fusion of 3D information for de novo drug design** [[Paper]](https://www.sciencedirect.com/science/article/pii/S1046202323000245) [[Code]](https://gitlab.com/cheminfIBB/pafnucy)
* [ACS 2023] **Universal Approach to De Novo Drug Design for Target Proteins Using Deep Reinforcement Learning** [[Paper]](https://pubs.acs.org/doi/full/10.1021/acsomega.2c06653)
* [ScienceDirect 2023] **Chemical language models for de novo drug design: Challenges and opportunities** [[Paper]](https://www.sciencedirect.com/science/article/pii/S0959440X23000015)
* [arXiv 2023] **Drug design on quantum computers** [[Paper]](https://arxiv.org/abs/2301.04114)
* [Nature Communications 2023] **Leveraging molecular structure and bioactivity with chemical language models for de novo drug design** [[Paper]](https://www.nature.com/articles/s41467-022-35692-6)
* [Int. J. Mol. Sci. 2023] **PETrans: De Novo Drug Design with Protein-Specific Encoding Based on Transfer Learning** [[Paper]](https://www.mdpi.com/1422-0067/24/2/1146) [[Code]](https://github.com/Chinafor/PETrans)
* [Wiley Interdisciplinary Reviews 2023] **Graph neural networks for conditional de novo drug design** [[Paper]](https://wires.onlinelibrary.wiley.com/doi/full/10.1002/wcms.1651)
* [Bioinformatics 2023] **HelixGAN a deep-learning methodology for conditional de novo design of Œ±-helix structures** [[Paper]](https://academic.oup.com/bioinformatics/article/39/1/btad036/6991169) [[Code]](https://github.com/xxiexuezhi/helix_gan)
* [arXiv 2023] **Fragment-based t-SMILES for de novo molecular generation** [[Paper]](https://arxiv.org/abs/2301.01829) [[Code]](https://github.com/juanniwu/t-smiles)
* [J. Chem. Inf. Model. 2023] **De Novo Direct Inverse QSPR/QSAR: Chemical Variational Autoencoder and Gaussian Mixture Regression Models** [[Paper]](https://pubs.acs.org/doi/full/10.1021/acs.jcim.2c01298) 
* [bioRxiv 2023] **A dual diffusion model enables 3D binding bioactive molecule generation and lead optimization given target pockets** [[Paper]](https://www.biorxiv.org/content/10.1101/2023.01.28.526011v1)
* [Bioinformatics 2023] **Sc2Mol: a scaffold-based two-step molecule generator with variational autoencoder and transformer** [[Paper]](https://academic.oup.com/bioinformatics/article/39/1/btac814/6964383) [[Code]](https://github.com/zhiruiliao/Sc2Mol)



2022
----
* [NeurIPS 2022] **Zero-Shot 3D Drug Design by Sketching and Generating**  [[Paper]](https://arxiv.org/abs/2209.13865) [[Code]](https://github.com/longlongman/DESERT)
* [arXiv 2022] **Equivariant Shape-Conditioned Generation of 3D Molecules for Ligand-Based Drug Design**  [[Paper]](https://arxiv.org/abs/2210.04893) [[Code]](https://github.com/keiradams/squid)
* [ICML 2022] **Generating 3D Molecules for Target Protein Binding** [[Paper]](https://arxiv.org/abs/2204.09410) [[Code]](https://github.com/divelab/GraphBP)
* [arXiv 2022] **Structure-based drug discovery with deep learning** | *Review*  [[Paper]](https://arxiv.org/abs/2212.13295) 
* [arXiv 2022] **DiffLinker: Equivariant 3D-Conditional Diffusion Model for Molecular Linker Design**  [[Paper]](https://arxiv.org/abs/2210.05274) [[Code]](https://github.com/igashov/DiffLinker)
* [ACS JCIM 2022] **HyFactor: A Novel Open-Source, Graph-Based Architecture for Chemical Structure Generation** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00744) [[Code]](https://github.com/Laboratoire-de-Chemoinformatique/HyFactor)
* [Journal of Cheminformatics 2022] **Designing optimized drug candidates with Generative Adversarial Network** [[Paper]](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-022-00623-6) [[Code]](https://github.com/larngroup/GAN-Drug-Generator)
* [arXiv 2022] **A Survey on Deep Graph Generation: Methods and Applications**  [[Paper]](https://arxiv.org/abs/2203.06714)
* [arXiv 2022] **Equivariant Diffusion for Molecule Generation in 3D** [[Paper]](https://arxiv.org/abs/2203.17003) [[Code]](https://github.com/ehoogeboom/e3_diffusion_for_molecules)
* [arXiv 2022] **Top-N: Equivariant set and graph generation without exchangeability** [[Paper]](https://arxiv.org/abs/2110.02096) [[Code]](https://github.com/cvignac/top-n)
* [ACS JCIM 2022] **RetroGNN: Fast Estimation of Synthesizability for Virtual Screening and De Novo Design by Learning from Slow Retrosynthesis Software** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.1c01476) 
* [ChemRxiv 2022] **Conditional ùõΩ-VAE for De Novo Molecular Generation** [[Paper]](https://chemrxiv.org/engage/chemrxiv/article-details/626b4332368ab64701913771)

2021
----
* [scientific report 2021] **Transformer neural network for protein-specific de novo drug generation as a machine translation problem** [[Paper]](https://www.nature.com/articles/s41598-020-79682-4)[[Code]](https://github.com/dariagrechishnikova/molecule_structure_generation)
* [RSC 2021] **Attention-based generative models for de novo molecular design**  [[Paper]](https://pubs.rsc.org/en/content/articlehtml/2021/sc/d1sc01050f)[[Code]](https://github.com/oriondollar/TransVAE)
* [Journal of Molecular Modeling 2021] **Generative chemistry: drug discovery with deep learning generative models** [[Paper]](https://arxiv.org/abs/2008.09000)
* [arXiv 2021] **A Graph VAE and Graph Transformer Approach to Generating Molecular Graphs** [[Paper]](https://arxiv.org/abs/2104.04345)
* [Journal of Chemical Information and Modeling 2021] **OpenChem: A Deep Learning Toolkit for Computational Chemistry and Drug Design** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.0c00971)[[Code]](https://github.com/Mariewelt/OpenChem)
* [arXiv 2021] **Transformers for Molecular Graph Generation** [[Paper]](https://www.esann.org/sites/default/files/proceedings/2021/ES2021-112.pdf)[[Code]](https://gitlab.uni-oldenburg.de/gies6280/molegent)
* [ACS JCIM 2021] **Exploring Graph Traversal Algorithms in Graph-Based Molecular Generation** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.1c00777)[[Code]](https://github.com/MolecularAI/GraphINVENT)
* [ACS JCIM 2021] **Generative Chemical Transformer: Neural Machine Learning of Molecular Geometric Structures from Chemical Language via Attention** [[Paper]](https://doi.org/10.1021/acs.jcim.1c01289)
* [ACS JCIM 2021] **De Novo Structure-Based Drug Design Using Deep Learning** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.1c01319) 



2020
----

* [RSC 2020] **Beyond Generative Models: Superfast Traversal, Optimization, Novelty, Exploration and Discovery (STONED) Algorithm for Molecules using SELFIES** [[Paper]](https://pubs.rsc.org/en/content/articlelanding/2021/sc/d1sc00231g)[[Code]](https://github.com/aspuru-guzik-group/stoned-selfies)
* [ChemRxiv 2020] **Comparative Study of Deep Generative Models on Chemical Space Coverage** [[Paper]](https://chemrxiv.org/engage/chemrxiv/article-details/60c755389abda285f4f8e2d1)[[Code]](https://github.com/jeah-z/Generative_Models_benchmark_gdb13)
* [CUoT 2020] **Comparison of State-of-the-art Algorithms for de novo Drug Design** [[Paper]](https://odr.chalmers.se/bitstream/20.500.12380/301739/1/CSE%2020-69%20Sundkvist%20Nilsson.pdf)[[Code]](https://github.com/sebastiandro/de-novo-evaluation)
* [arXiv 2020] **Deep Molecular Dreaming: Inverse machine learning for de-novo molecular design and interpretability with surjective representations** [[Paper]](https://www.arxiv-vanity.com/papers/2012.09712/)
* [Nature Machine Intelligence 2020] **Direct steering of de novo molecular generation with descriptor conditional recurrent neural networks** [[Paper]](https://www.nature.com/articles/s42256-020-0174-5)[[Code]](https://github.com/pcko1/Deep-Drug-Coder/tree/master/datasets)
* [arxiv 2020] **Generating 3D Molecular Structures Conditional on a Receptor Binding Site with Deep Generative Models** [[Paper]](https://arxiv.org/abs/2010.14442)
* [Drug Discovery Today: Technologies 2020] **Graph-based generative models for de Novo drug design** [[Paper]](https://www.sciencedirect.com/science/article/pii/S1740674920300251)
* [arXiv 2020] **MolDesigner: Interactive Design of Efficacious Drugs with Deep Learning** [[Paper]](https://arxiv.org/abs/2010.03951)
* [ChemRxiv 2020] **Practical Notes on Building Molecular Graph Generative Models** [[Paper]](https://chemrxiv.org/engage/chemrxiv/article-details/60c74f55567dfe705bec5672)[[Code]](https://github.com/MolecularAI/GraphINVENT)
* [arXiv 2020] **RetroGNN: Approximating Retrosynthesis by Graph Neural Networks for De Novo Drug Design** [[Paper]](https://arxiv.org/abs/2011.13042)
* [arXiv 2020] **Seq2Mol: Automatic design of de novo molecules conditioned by the target protein sequences through deep neural networks** [[Paper]](https://arxiv.org/abs/2010.15900)
* [arXiv 2020] **Target-specific and selective drug design for covid-19 using deep generative models** [[Paper]](https://arxiv.org/abs/2004.01215)
* [ChemRxiv 2020] **REINVENT 2.0 ‚Äì an AI Tool for De Novo Drug Design** [[Paper]](https://chemrxiv.org/engage/chemrxiv/article-details/60c74f75bdbb89eaf7a39d8a)[[Code]](https://github.com/MolecularAI/Reinvent)
* [Front. Pharmacol. 2020] **Molecular Generation for Desired Transcriptome Changes With Adversarial Autoencoders** [[Paper]](https://www.frontiersin.org/articles/10.3389/fphar.2020.00269/full)[[Code]](https://github.com/insilicomedicine/BiAAE)
* [ACS 2020] **The Synthesizability of Molecules Proposed by Generative Models** [[Paper]](https://pubs.acs.org/doi/full/10.1021/acs.jcim.0c00174)[[Code]](https://github.com/wenhao-gao/askcos_synthesizability)

2019
----

* [Journal of Cheminformatics 2019] **A de novo molecular generation method using latent vector based generative adversarial network**  [[Paper]](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-019-0397-9)[[Code]](https://github.com/Dierme/latent-gan)
* [Nature Machine Intelligence 2019] **Automated de novo molecular design by hybrid machine intelligence and rule-driven chemical synthesis** [[Paper]](https://www.nature.com/articles/s42256-019-0067-7)
* [arXiv 2019] **ChemBO: Bayesian Optimization of Small Organic Molecules with Synthesizable Recommendations** [[Paper]](https://arxiv.org/abs/1908.01425)[[PapersWithCode]](https://paperswithcode.com/paper/chembo-bayesian-optimization-of-small-organic)
* [Journal of Chemical Information and Modeling 2019] **Conditional Molecular Design with Deep Generative Models** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.8b00263)[[Code]](https://github.com/nyu-dl/conditional-molecular-design-ssvae)
* [Journal of Chemical Information and Modeling 2019] **De Novo Molecular Design by Combining Deep Autoencoder Recurrent Neural Networks with Generative Topographic Mapping** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.8b00751)
* [arXiv 2019] **Deep learning for molecular design‚Äîa review of the state of the art** [[Paper]](https://arxiv.org/abs/1903.04388)
* [Future medicinal chemistry 2019] **Deep learning for molecular generation** [[Paper]](https://pubmed.ncbi.nlm.nih.gov/30698019/)
* [Journal of chemical information and modeling 2019] **Drug Analogs from Fragment-Based Long Short-Term Memory Generative Neural Networks** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.8b00902)
* [RSC 2019] **Efficient Multi-Objective Molecular Optimization in a Continuous Latent Space** [[Paper]](https://pubs.rsc.org/en/content/articlelanding/2019/sc/c9sc01928f)[[Code]](https://github.com/jrwnter/mso)
* [Journal of Cheminformatics 2019] **Exploring the GDB-13 chemical space using deep generative models** [[Paper]](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-019-0341-z)[[Code]](https://github.com/undeadpixel/reinvent-gdb13)
* [arXiv 2019] **Likelihood-Free Inference and Generation of Molecular Graphs** [[Paper]](https://arxiv.org/pdf/1905.10310.pdf)[[Code]](https://github.com/ai-med/almgig)
* [arXiv 2019] **A Model to Search for Synthesizable Molecules** [[Paper]](https://arxiv.org/abs/1906.05221)[[Code]](https://github.com/john-bradshaw/molecule-chef)
* [arXiv 2019] **MolecularRNN: Generating realistic molecular graphs with optimized properties** [[Paper]](https://arxiv.org/abs/1905.13372)[[PapersWithCode]](https://paperswithcode.com/paper/molecularrnn-generating-realistic-molecular)
* [Journal of Cheminformatics 2019] **Randomized SMILES Strings Improve the Quality of Molecular Generative Models** [[Paper]](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-019-0393-0)[[Code]](https://github.com/molecularsets/moses)
* [arXiv 2019] **Scaffold-based molecular design using graph generative model** [[Paper]](https://arxiv.org/abs/1905.13639)
* [ACS 2019] **Shape-Based Generative Modeling for de Novo Drug Design** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.8b00706)[[Code]](https://github.com/compsciencelab/ligdream)
* [arXiv 2019] **A Two-Step Graph Convolutional Decoder for Molecule Generation** [[Paper]](https://arxiv.org/abs/1906.03412)

2018
----

* [ACS Central Science 2018] **Automatic Chemical Design Using a Data-Driven Continuous Representation of Molecules** [[Paper]](https://pubs.acs.org/doi/10.1021/acscentsci.7b00572)[[Code]](https://github.com/aspuru-guzik-group/chemical_vae)
* [Science Advances 2018] **Deep reinforcement learning for de novo drug design** [[Paper]](https://www.science.org/doi/10.1126/sciadv.aap7885)[[Code]](https://github.com/isayev/ReLeaSE)
* [ACS JCIM 2018] **Fr√©chet ChemNet Distance: A Metric for Generative Models for Molecules in Drug Discovery** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.8b00234)[[Code]](https://github.com/bioinf-jku/FCD)
* [arXiv 2018] **MolGAN: An implicit generative model for small molecular graphs** [[Paper]](https://arxiv.org/abs/1805.11973)[[PapersWithCode]](https://paperswithcode.com/paper/molgan-an-implicit-generative-model-for-small)
* [Journal of Cheminformatics 2018] **Multi-objective de novo drug design with conditional graph generative model** [[Paper]](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-018-0287-6)[[Code]](https://github.com/kevinid/molecule_generator)
* [ACS Medicinal Chemistry Letters 2018] **Transforming Computational Drug Discovery with Machine Learning and AI** [[Paper]](https://pubs.acs.org/doi/10.1021/acsmedchemlett.8b00437)
* [Journal of chemical information and modeling 2018] **Sparse Generative Topographic Mapping for Both Data Visualization and Clustering** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.8b00528)[[Code]](https://github.com/hkaneko1985/gtm-generativetopographicmapping)
* [Journal of Cheminformatics 2018] **Molecular generative model based on conditional variational autoencoder for de novo molecular design** [[Paper]](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-018-0286-7) [[Code]](https://github.com/jaechanglim/CVAE)


2017
----

* [Molecular Pharmaceutics 2017] **druGAN: An Advanced Generative Adversarial Autoencoder Model for de Novo Generation of New Molecules with Desired Molecular Properties in Silico** [[Paper]](https://pubs.acs.org/doi/10.1021/acs.molpharmaceut.7b00346)
* [ACS Central Science 2017] **Generating Focused Molecule Libraries for Drug Discovery with Recurrent Neural Networks** [[Paper]](https://pubs.acs.org/doi/10.1021/acscentsci.7b00512)
* [Journal of Cheminformatics 2017] **Molecular de-novo design through deep reinforcement learning** [[Paper]](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-017-0235-x)[[Code]](https://github.com/MarcusOlivecrona/REINVENT)

Acknowledgement
----

- Papers between 2017-2021 about **De novo Drug Design :pill:** collected by [@HUBioDataLab](https://github.com/HUBioDataLab) members while I was in research intern program.

- Template by [@mengliu1998](https://github.com/mengliu1998)!